Modules/IO/JPEG/src/itkJPEGImageIO.cxx

```diff
@@ -247,7 +247,7 @@ JPEGImageIO::JPEGImageIO()
 #if BITS_IN_JSAMPLE == 8
   m_ComponentType = IOComponentEnum::UCHAR;
 #else
-  m_ComponentType = IOComponentEnum::UINT;
+#  error
```

Comment:
We should probably add a [message](https://en.cppreference.com/w/cpp/preprocessor/error) here. Maybe "JPEG files with more than 8 bits per sample are not supported"?

Reply:
Done.